fix:  APP-737 empty state email profile

alexander-astrand opened the pull request:
## Description

https://regennetwork.atlassian.net/browse/APP-737

<!-- Add a description of the changes that this PR introduces and the files that
are the most critical to review. -->

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

<!-- This should include steps on how to test the fixes or features within the marketplace app and/or the website.
If applicable, this should also include links to the created/updated components on storybook. -->

1. Log in with email that has no projects and go to public profile. Make sure padding is correct on mobile and project tab shows when both logged in and out.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

r41ph commented on web-marketplace/src/pages/Profile/ProjectsTab/NoProjectsDisplay.tsx:
```diff
@@ -18,7 +18,7 @@ export const NoProjectsDisplay = ({
     <div
       className={`flex flex-col justify-center items-center rounded-[10px]
-                  h-[321px] px-5 gap-[15px] py-[40px]
+                  h-[321px] px-5 gap-[15px] md:py-[40px] -mt-[30px] md:mt-0
```
nit: 
```suggestion
                  h-[321px] px-5 gap-15 md:py-40 -mt-30 md:mt-0
```

alexander-astrand commented:
@erikalogie @S4mmyb See testing instructions!

alexander-astrand commented:
> > > Looks good, I think we should just tighten up this spacing so it is more like it is in the comps: <img alt="Monosnap Regen Marketplace : Buy, trade and retire carbon credits | Regen Marketplace : Buy, trade and retire carbon credits 2025-08-26 12-38-35" width="399" height="629" src="https://private-user-images.githubusercontent.com/8460237/482208761-83f20bd9-a685-48ff-b9e4-fef763a96f5b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY3MjI3MDcsIm5iZiI6MTc1NjcyMjQwNywicGF0aCI6Ii84NDYwMjM3LzQ4MjIwODc2MS04M2YyMGJkOS1hNjg1LTQ4ZmYtYjllNC1mZWY3NjNhOTZmNWIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MDFUMTAyNjQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjJjYWNlNWE1YTUwMTZhMjBjMWE2NDJhNmIxNTFiM2EyMWVkMmIwMDczMzMyYWMyNTZmNGUyNGFhMGU4NWRkMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.M3VCUNf7SJzd2laMV3n7StQBUxEXbQlLNOxL5oQJ2w4">
> > 
> > 
> > I added the email address so it is visible when logged in with email now, and tightened the space when logged out. The space above the tabs, do we want that tightened always on both mobile and deskop and for email, wallet etc? Right now the padding is 60px above the tabs always.
> 
> Should be 30px on mobile. I'm not sure that we want to show their email address here, as that might be confusing to the user. They may think that it shows on the public profile, which it should not of course because of privacy. I would just have it look like on the logged out version with the tightened up space.

This should be sorted now! I removed the email again from the logged in state.
